### ISSUE TITLE
barcody.bas won't encode numbers

### ISSUE THREAD

**Issue by makulais** [witwall/barcode-vba-macro-only#4]
If a number with no characters included in it is input as the string to be encoded, the QR gen file does not encode any value at all into the QR code. Tested with values of 0 - 65535, ... no value is encoded into the QR code. Tested with all ECL's, no value is encoded into the QR code. If a single character is included into the string, then the QR code will correctly implement the string, but if it is strictly numbers it will not.

**Comment by JonasHeidelberg** [comment]
Good catch! I might take some time over Christmas to look into this. If you need it quicker/want to get active yourself, I am sure it is in the same part of the code as the previous bugs I fixed (see https://github.com/jonasheidelberg/barcode-vba-macro-only, function qr_gen() in barcody.bas)

**Comment by JonasHeidelberg** [comment]
Hi, actually [my fork](https://github.com/JonasHeidelberg/barcode-vba-macro-only/) had already fixed this bug for inputs from 10-65535 three weeks ago. I now also fixed it for single-digit numbers - you can find the updated code here: https://github.com/JonasHeidelberg/barcode-vba-macro-only/commits/quickfix-witwall-issue-4/barcody.bas 
Note you need to download the barcody.bas file - the XLSX is not updated yet. Hope this helps!

**Comment by JonasHeidelberg** [comment]
... I now also updated the XLSX and merged into my Master https://github.com/JonasHeidelberg/barcode-vba-macro-only
Would love to hear if this solves your issue!

**Comment by makulais** [comment]
Worked great, I thought it might be an off-by-one, but it would have taken
me forever to find it. Would love an explanation of how his code is
working! Thank you. My company decided to go ahead and purchase a dll for
use as we were concerned about bugs showing up during production, but I'll
keep this as a back up in case we ever lose that driver due to upgrades,
issues, ....

Thanks again, nice job.

Kennard

**Comment by JonasHeidelberg** [comment]
Glad to hear you liked it! For commercial production code I completely agree with your company‘s decision! Until someone has made a somewhat extensive code review, this thing is a nice-to-have where you should always check that the output is what you expect...

**Comment by kazu-kazu** [comment]
QR gen
If a comma is included in the number
It cannot be recognized correctly.
Please..


Example: In excel
555,0123456789,550,9999999999

**Comment by harisan168** [comment]
Add a CHR(13) or CHR(10) at the end your input text to encode it correctly with numeric value.
